go/ct/spc/specification.go

```diff
@@ -249,7 +249,13 @@ func getAllRules() []Rule {
 			}
 			s.Gas -= memExpCost
 
-			wordCost := vm.Gas(6 * ((size + 31) / 32))
+			words, overflow := wordCountWithOverflow(size)
+			if overflow {
+				s.Status = st.Failed
+				s.Gas = 0
+				return
+			}
+			wordCost := vm.Gas(6 * words)
```

Comment:
As part of the `ExpansionCosts` function the gas costs for the expansion is computed. If this cost [overflows](https://github.com/Fantom-foundation/Tosca/blob/7d737ec9f5f83e4e53b7b8c73cf292fdba1cecde/go/ct/st/memory.go#L108), `MaxUint64` is returned for the expansion cost, which always exceeds the amount of available gas (since the static costs of this operation are not 0).

Thus, technically this `wordCost` computation was guarded against overflows, but it was not obvious. I agree, this change makes this more apparent, but I would suspect that the `overflow` branch is dead code.